core/rawdb/bor_receipt.go

```diff
@@ -9,15 +11,19 @@ import (
 )
 
 var (
-	borReceiptPrefix = []byte("bor-receipt-") // borReceiptPrefix + number + block hash -> bor block receipt
+	// bor receipt key
+	borReceiptKey = types.BorReceiptKey
+
+	// borTxLookupPrefix + hash -> transaction/receipt lookup metadata
+	borTxLookupPrefix = []byte("matic-bor-tx-lookup-")
```

Comment:
Does it denote, we can query state sync log using `append([]byte("matic-bor-tx-lookup-"), hash.Bytes()...)` , where `hash` is bor internal ( i.e. system call ) tx hash 🤔 ?

Reply:
Yes, this is a reverse lookup to get the block number for the given bor receipt hash.